### DISCUSSION TITLE
Remove format

### DISCUSSION THREAD

**Discussion by fedot199614** [samclarke/SCEditor#923] [Q&A]
is it possible to remove text formatting by default in case when i copy - paste some text
@samclarke pls, advice

**Comment by samclarke** [comment]
There is a [plain text plugin](https://www.sceditor.com/documentation/plugins/plaintext/) which will remove the formatting on anything pasted into the editor.

The plugin won't stop the formatting from being copied, so if the copied text is pasted into another editor, the formatting will still exist.
